config/registry/aws/modules/lambda-function.yaml

```diff
@@ -0,0 +1,68 @@
+halt: false
+environment_module: false
+inputs:
+  - name: env_name
+    user_facing: false
+    description: Opta Environment name
+    default: None
+  - name: layer_name
+    user_facing: false
+    description: Opta Layer name
+    default: None
+  - name: module_name
+    user_facing: false
+    description: Opta Module name
+    default: None
+  - name: extra_iam_policies
+    user_facing: true
+    validator: list(str(), required=False)
+    description: The arns of additional IAM policies to be attached to this role.
+    default: []
+  - name: expose_via_domain
+    user_facing: true
+    validator: bool(required=False)
+    description: The arns of additional IAM policies to be attached to this role.
+    default: false
+  - name: runtime
+    user_facing: true
+    validator: str(required=False)
+    description: The AWS lambda runtime to use for this lambda (see https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html)
+    default: "nodejs14.x"
+  - name: filename
+    user_facing: true
+    validator: str(required=False)
+    description: The filename for your zip file containing your lambda code
+    default: null
+  - name: env_vars
+    user_facing: true
+    validator: map(str(), str(), required=False)
+    description: |
+      A map of key values to add to the container as environment variables (key is name, value is value).
+      ```yaml
+      env_vars:
+       FLAG: "true"
+      ```
+    default: { }
+  - name: vpc_id
+    user_facing: false
+    description: Opta vpc_id id
```

Comment:
Does this need to be an opta vpc or could it be any vpc?

Reply:
It needs to be the opta vpc for now because of the way we find the subnets for it. Not opposed to finding a way to make it work with non-opta vpcs in the future, but that's beyond anything we talked about for v1 (also note that this input is hidden from the user--it's automatically filled out if opta detects that an aws-base module is present)